recipes/beauty/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,7 @@
+#include <beauty/beauty.hpp>
+
+int main()
+{
+    // Create a server
+    beauty::server server;
```

Comment:
in general, it's preferred to actually call some functions, because unused objects are subject for the removal by compiler (even if they have side-effects inside ctor). but it's fine to leave for now, no need to edit.

Reply:
Ok. I note it